src/hope/apps/periodic_data_update/celery_tasks.py

```diff
@@ -134,3 +136,24 @@ def remove_old_pdu_template_files_task(self: Any, expiration_days: int = 30) ->
     except Exception as e:  # pragma: no cover
         logger.exception("Remove old PDU files Error")  # pragma: no cover
         raise self.retry(exc=e)  # pragma: no cover
+
+
+@app.task(bind=True, default_retry_delay=60, max_retries=3)
+@log_start_and_end
+@sentry_tags
+def send_pdu_online_edit_notification_emails(
+    self: Any,
+    pdu_online_edit_id: int,
+    action: str,
+    action_user_id: str,
+    action_date_formatted: str,
+) -> None:
+    from hope.apps.periodic_data_update.notifications import PDUOnlineEditNotification
+
+    try:
+        pdu_online_edit = PDUOnlineEdit.objects.get(id=pdu_online_edit_id)
+        action_user = User.objects.get(id=action_user_id)
+        set_sentry_business_area_tag(pdu_online_edit.business_area.name)
+        PDUOnlineEditNotification(pdu_online_edit, action, action_user, action_date_formatted).send_email_notification()
+    except Exception as e:
+        logger.exception(e)
```

Comment:
logger.exception() automatically includes traceback so logger.exception("message") not logger.exception(e)